templates/user/notification/notification.tmpl

```diff
@@ -14,6 +14,14 @@
 			<a href="{{AppSubUrl}}/notifications?q=read" class="{{if eq .Status 2}}active{{end}} item">
 				{{.i18n.Tr "notification.read"}}
 			</a>
+			{{if and (gt (len .Notifications) 0) (eq .Status 1) (.NotificationUnreadCount)}}
```

Comment:
Is check for `gt (len .Notifications) 0` really needed?

Reply:
I felt as though it should only show if there are unread notifications -- otherwise, the mark all as read wouldn't do anything as there's nothing to mark

Reply:
Yes but there is NotificationUnreadCount check